develop/dev-guide-create-table.md

```diff
@@ -102,7 +102,7 @@ A [primary key](/constraints.md#primary-key) is a column or a set of columns in
 
 > **Note:**
 >
-> The default definition of **primary key** in TiDB is different from that in [InnoDB](https://mariadb.com/kb/en/innodb/)(the common storage engine of MySQL). 
+> The default definition of **primary key** in TiDB is different from that in [InnoDB](https://mariadb.com/kb/en/innodb/)(the common storage engine of MySQL).
```

Comment:
```suggestion
> The default definition of **primary key** in TiDB is different from that in [InnoDB](https://dev.mysql.com/doc/refman/8.0/en/innodb-storage-engine.html)(the common storage engine of MySQL).
```

Reply:
Looks like this same sentence and links are also in `develop/dev-guide-schema-design-overview.md`

Reply:
Thank you! Just fixed it.